devtools/mog/generate.go

```diff
@@ -213,12 +224,155 @@ func generateToFunc(cfg structConfig, imports *imports) *ast.FuncDecl {
 	}
 }
 
-func funcNameTo(cfg structConfig) string {
-	return cfg.Source + "To" + cfg.FuncNameFragment
+func generatePtrToPtrFunc(cfg structConfig, imports *imports) *ast.FuncDecl {
```

Comment:
I suspect these variations would be slightly less absurd if we bootstrapped them off of pre-baked strings using:
```
	fset := token.NewFileSet()
	f, err := parser.ParseFile(fset, "", src, 0)
```

somehow